### PR TITLE
Add an SSL setup guide

### PR DESCRIPTION
**Pull request by d-513**
This also fixes websockets in the non ssl nginx config example

### CODE REVIEW

File: source/guides/ssl-setup-nginx.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+##################
+
+The following ports are used by nginx. Please allow traffic to/from these ports.
+
+* 80: HTTP traffic.
+* 443: HTTPS traffic.
+
+To set up SSL with let's encrypt you will also need a domain pointed to your panel's IP address.
+
+Setting up pufferpanel to be served over SSL is simple. We will use nginx and an Let's Encrypt certificate.
```
Comment (LordRalex) [inline source/guides/ssl-setup-nginx.rst]: PufferPanel

File: source/guides/ssl-setup-nginx.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                 proxy_set_header X-Nginx-Proxy true;
+                 proxy_http_version 1.1;
+                 proxy_set_header Upgrade $http_upgrade;
+                 proxy_set_header Connection "Upgrade";
+                 proxy_set_header Host $host;
+             }
+         }
+
+Click CTRL+x to save, and then press Y and enter to confirm.
+Restart nginx. You should be able to access pufferpanel at the domain you've choosen above.
```
Comment (LordRalex) [inline source/guides/ssl-setup-nginx.rst]: PufferPanel

File: source/guides/ssl-setup-nginx.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+   .. tab:: Ubuntu/Debian
+
+      .. code-block:: bash
+
+         sudo apt-get update
+         sudo apt-get install nginx
+         
+Now navigate to http://yourip. You should see a default nginx page.  
+
+Now set up an nginx virtual host. This will allow nginx to serve pufferpanel.
```
Comment (LordRalex) [inline source/guides/ssl-setup-nginx.rst]: PufferPanel

**Review by LordRalex — CHANGES_REQUESTED** [review]

File: source/guides/ssl-setup-nginx.rst (hunk 3 above)
Comment (d-513) [reply source/guides/ssl-setup-nginx.rst]: Fixed

**Pull request merged**